templates/_http.conf.erb

```diff
@@ -1 +1,2 @@
 ProxyPass /unattended http://localhost:3000/unattended
+Redirect permanent <%= scope['katello_devel::deployment_url'] %>/api https://<%= scope['katello_devel::fqdn'] %><%= scope['katello_devel::deployment_url'] %>/api
```

Comment:
Why do we want to limit this strictly to the katello API paths? Versus just doing the whole application?